formwidgets/mlmarkdowneditor/assets/js/mlmarkdowneditor.js

```diff
@@ -37,25 +39,28 @@
     MLMarkdownEditor.DEFAULTS = {
         textareaElement: null,
         placeholderField: null,
+        copyHandler: null,
         defaultLocale: 'en'
     }
 
     MLMarkdownEditor.prototype.init = function() {
         this.$el.multiLingual()
 
         this.$el.on('setLocale.oc.multilingual', this.proxy(this.onSetLocale))
+        this.$el.on('copyLocale.oc.multilingual', this.proxy(this.onCopyLocale))
         this.$textarea.on('changeContent.oc.markdowneditor', this.proxy(this.onChangeContent))
 
         this.updateLayout()
+        this.initInputEvents()
```

Comment:
@IsaiahPaget could you make sure these events are disposed of in the dispose function as well?